lib/Pipeline/Context.cpp

```diff
@@ -44,6 +44,7 @@ llvm::Error Context::load(const revng::DirectoryPath &Path) {
   if (not MaybeExists)
     return MaybeExists.takeError();
 
+  // TODO: this is suspicious, we should probably add a comment why.
```

Comment:
The check below is basically `stat`, so if the file doesn't exist we stop and succeed. Either way a comment doesn't hurt.

Reply:
Yeah, that's what I mean, it's suspicious for a _load_ function to succeed when the file it's loading _from_ doesn't exist.

Reply:
I see. I'll add a comment!

Reply:
Done:
```
  // In the context of `lib/Pipeline`, both context and containers are default
  // constructible. As such, we just skip loading files that do not exist,
  // ending up with the default state.
```